tasks/acquia.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+tasks:
+  auth:
+    cmds:
+      - if [ -z "$ACQUIA_API_KEY" ]; then echo "ACQUIA_API_KEY is empty, please add it to your .env file"; exit 1; fi
+      - if [ -z "$ACQUIA_API_SECRET" ]; then echo "ACQUIA_API_SECRET is empty, please add it to your .env file"; exit 1; fi
+      - echo "💧 Authorising with Acquia"
+      - acli auth:login --no-interaction --key=${ACQUIA_API_KEY} --secret=${ACQUIA_API_SECRET}
```

Comment:
One thought: Creating a full login session with Pantheon is slow. Is this command slow too? We call this every time we fetch a database below, and I'm wondering if there's some fast way to know we have a valid session so we don't have to do that every time.

Reply:
I've tried a similar one many times and didn't feel the command was slow.